<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorMaterial</class>
 <widget class="QWidget" name="InspectorMaterial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Material</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Material</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="19" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Tiling</string>
        </property>
       </widget>
      </item>
      <item row="19" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_pTilingXEdit">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_pTilingYEdit">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Value</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Texture</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSlider" name="m_pRoughnessSlider">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_pRoughnessEdit">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_9">
        <property name="styleSheet">
         <string notr="true">font-weight:bold;</string>
        </property>
        <property name="text">
         <string>Metallic</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Texture</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font-weight:bold;</string>
        </property>
        <property name="text">
         <string>Normal</string>
        </property>
       </widget>
      </item>
      <item row="16" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QSlider" name="m_pReflectanceSlider">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::NoTicks</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_pReflectanceEdit">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="20" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_pOffsetXEdit">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_pOffsetYEdit">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font-weight:bold;</string>
        </property>
        <property name="text">
         <string>Albedo</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="m_pNormalTextureEdit"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="m_pAlbedoTextureEdit"/>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">font-weight:bold;</string>
        </property>
        <property name="text">
         <string>Roughness</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLineEdit" name="m_pRoughnessTextureEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Texture</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Texture</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="m_pAlbedoColorButton">
          <property name="palette">
           <palette>
            <active>
             <colorrole role="Button">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="Button">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="Button">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_pAlbedoPickButton">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Pick</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="13" column="1">
       <widget class="QLineEdit" name="m_pMetallicTextureEdit"/>
      </item>
      <item row="16" column="0">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>Value</string>
        </property>
       </widget>
      </item>
      <item row="14" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QSlider" name="m_pMetallicSlider">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_pMetallicEdit">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="20" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Offset</string>
        </property>
       </widget>
      </item>
      <item row="15" column="0">
       <widget class="QLabel" name="label_10">
        <property name="styleSheet">
         <string notr="true">font-weight:bold;</string>
        </property>
        <property name="text">
         <string>Reflectance</string>
        </property>
       </widget>
      </item>
      <item row="18" column="0">
       <widget class="QLabel" name="label_12">
        <property name="styleSheet">
         <string notr="true">font-weight:bold;</string>
        </property>
        <property name="text">
         <string>Tiling &amp; Offset</string>
        </property>
       </widget>
      </item>
      <item row="14" column="0">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>Value</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_pAlbedoTextureEdit</tabstop>
  <tabstop>m_pAlbedoColorButton</tabstop>
  <tabstop>m_pAlbedoPickButton</tabstop>
  <tabstop>m_pNormalTextureEdit</tabstop>
  <tabstop>m_pRoughnessTextureEdit</tabstop>
  <tabstop>m_pRoughnessSlider</tabstop>
  <tabstop>m_pRoughnessEdit</tabstop>
  <tabstop>m_pMetallicTextureEdit</tabstop>
  <tabstop>m_pMetallicSlider</tabstop>
  <tabstop>m_pMetallicEdit</tabstop>
  <tabstop>m_pReflectanceSlider</tabstop>
  <tabstop>m_pReflectanceEdit</tabstop>
  <tabstop>m_pTilingXEdit</tabstop>
  <tabstop>m_pTilingYEdit</tabstop>
  <tabstop>m_pOffsetXEdit</tabstop>
  <tabstop>m_pOffsetYEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_pAlbedoPickButton</sender>
   <signal>clicked()</signal>
   <receiver>InspectorMaterial</receiver>
   <slot>pickColorFromDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>547</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>587</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pAlbedoColorButton</sender>
   <signal>clicked()</signal>
   <receiver>InspectorMaterial</receiver>
   <slot>pickColorFromDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>477</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>593</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pAlbedoTextureEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>InspectorMaterial</receiver>
   <slot>valueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>593</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pNormalTextureEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>InspectorMaterial</receiver>
   <slot>valueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>473</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>589</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pRoughnessTextureEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>InspectorMaterial</receiver>
   <slot>valueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>591</x>
     <y>398</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pMetallicTextureEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>InspectorMaterial</receiver>
   <slot>valueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>595</x>
     <y>476</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pTilingXEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>InspectorMaterial</receiver>
   <slot>valueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>731</y>
    </hint>
    <hint type="destinationlabel">
     <x>595</x>
     <y>668</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pTilingYEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>InspectorMaterial</receiver>
   <slot>valueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>743</y>
    </hint>
    <hint type="destinationlabel">
     <x>591</x>
     <y>630</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pOffsetXEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>InspectorMaterial</receiver>
   <slot>valueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>785</y>
    </hint>
    <hint type="destinationlabel">
     <x>591</x>
     <y>714</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pOffsetYEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>InspectorMaterial</receiver>
   <slot>valueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>471</x>
     <y>779</y>
    </hint>
    <hint type="destinationlabel">
     <x>591</x>
     <y>576</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pRoughnessEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>InspectorMaterial</receiver>
   <slot>valueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>593</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pMetallicEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>InspectorMaterial</receiver>
   <slot>valueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>544</y>
    </hint>
    <hint type="destinationlabel">
     <x>599</x>
     <y>514</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pReflectanceEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>InspectorMaterial</receiver>
   <slot>valueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>638</y>
    </hint>
    <hint type="destinationlabel">
     <x>591</x>
     <y>606</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pRoughnessSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorMaterial</receiver>
   <slot>roughnessValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>595</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pMetallicSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorMaterial</receiver>
   <slot>metallicValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>413</x>
     <y>550</y>
    </hint>
    <hint type="destinationlabel">
     <x>595</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pReflectanceSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorMaterial</receiver>
   <slot>reflectanceValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>640</y>
    </hint>
    <hint type="destinationlabel">
     <x>589</x>
     <y>766</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pickColorFromDialog()</slot>
  <slot>valueChanged()</slot>
  <slot>roughnessValueChanged(int)</slot>
  <slot>metallicValueChanged(int)</slot>
  <slot>reflectanceValueChanged(int)</slot>
 </slots>
</ui>
